code/game/objects/items/weapons/material/shield_vr.dm

```diff
@@ -0,0 +1,16 @@
+/obj/item/weapon/material/sword/shield
```

Comment:
why does it specifically have to draw from the sword subtype of material weapons? define a new type, e.g.  `/obj/item/weapon/material/shield`